extensions/interactions/rulesSpec.js

```diff
@@ -26,7 +26,7 @@ describe('Rule spec services', function() {
 
   var getRulesServiceName = function(interactionId) {
     return (
-      interactionId[0].toLowerCase() + interactionId.substr(1) + 'RulesService'
+      interactionId[0] + interactionId.substr(1) + 'RulesService'
```

Comment:
The first two items here could just be replaced with `interactionId`, right?

Reply:
done